pom.xml

```diff
@@ -23,7 +23,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-        <serve.version>7.4.1</serve.version>
+        <serve.version>8.0.0-beta.1</serve.version>
```

Comment:
Why is the SERVE release a beta? We have tested this properly, would have assumed it would be a proper release.